tests/models/depth_anything/test_modeling_depth_anything.py

```diff
@@ -265,4 +263,23 @@ def test_inference(self):
             [[8.8204, 8.6468, 8.6195], [8.3313, 8.6027, 8.7526], [8.6526, 8.6866, 8.7453]],
         ).to(torch_device)
 
-        self.assertTrue(torch.allclose(outputs.predicted_depth[0, :3, :3], expected_slice, atol=1e-6))
+        self.assertTrue(torch.allclose(predicted_depth[0, :3, :3], expected_slice, atol=1e-6))
+
+        # -- `metric` depth model
+        # TODO: change path to `depth-anything/depth-anything-V2-metric-indoor-small-hf` once it's available
```

Comment:
What's pending for the checkpoint to be made available?

Reply:
I just don't know how to convert the checkpoints under the organization name 😅 Could you please provide some guidance for this? When I run the [convert_depth_anything_to_hf.py](https://github.com/huggingface/transformers/blob/f0bc49e7f61f74f055c47ad40e6010f57eed0b0b/src/transformers/models/depth_anything/convert_depth_anything_to_hf.py) script, it pushes the checkpoints on the hub under my workspace

Reply:
I can transfer them from your HF profile to the organization

Reply:
Thanks @NielsRogge! Here's the list of checkpoints to transfer:
-  `bthia97/Depth-Anything-V2-Metric-Indoor-Small-hf`
- `bthia97/Depth-Anything-V2-Metric-Indoor-Base-hf`
- `bthia97/Depth-Anything-V2-Metric-Indoor-Large-hf`
- `bthia97/Depth-Anything-V2-Metric-Outdoor-Small-hf`
- `bthia97/Depth-Anything-V2-Metric-Outdoor-Base-hf`
- `bthia97/Depth-Anything-V2-Metric-Outdoor-Large-hf`

Am I in charge of designing the model cards? Please let me know if I need to do it! :)

Reply:
Thanks, I can take care of this tonight. Feel free to add model cards

Reply:
Done! I took example on the cards you created for Depth Anything V2 (relative depth metric) :)

Reply:
Thanks, just transferred them all :)